src/pages/ProductsPage/ProductCard/ProductCardComponent.tsx

```diff
@@ -0,0 +1,132 @@
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import DescriptionIcon from '@mui/icons-material/Description';
+import Box from '@mui/material/Box';
+import * as styles from "./styles"
```

Comment:
you can simplify it: import { styles } from './styles';

Reply:
I`ve added a new logic to styles.ts file to simplify this import